src/Bicep.Core/Configuration/bicepconfig.json

```diff
@@ -16,17 +15,11 @@
             "login.microsoftonline.com",
             "graph.windows.net",
             "trafficmanager.net",
-            "vault.azure.net",
             "datalake.azure.net",
             "azuredatalakestore.net",
             "azuredatalakeanalytics.net",
             "vault.azure.net",
             "api.loganalytics.io",
-            "api.loganalytics.iov1",
-            "asazure.windows.net",
-            "region.asazure.windows.net",
-            "api.loganalytics.iov1",
-            "api.loganalytics.io",
             "asazure.windows.net",

```

Comment:
Not sure why, but there were a bunch of duplicates here, and `api.loganalytics.iov1` looks like a typo. I've removed them.